blogs/artificial-intelligence/mamba2-flash-attention-kernels/README.md

```diff
@@ -0,0 +1,113 @@
+---
+blogpost: true
+date: 07 December 2024
+author: Quentin Anthony, Beren Millidge
+tags: LLM, AI/ML, GenAI, PyTorch
+category: Applications & models
+language: English
+myst:
+  html_meta:
+    "description lang=en": "Mamba2 and Flash Attention Backward Kernels on AMD MI300x with ROCm"
+    "keywords": "Mamba, PyTorch, S4, S6, Mamba2, Transformer, Flash Attention, Optimization, Hardware-aware, Transformer, Attention, ROCm, Mi210, MI250, MI300, AI/ML, Generative AI"
+    "property=og:locale": "en_US"
+---
+
+
+# Training Transformers and Hybrid models on AMD Instinct MI300X Accelerators
+
+*By Quentin Anthony and Beren Millidge from Zyphra*
+
+On paper, the AMD Instinct MI300X GPU accelerators contain some of the best hardware specifications on the market, significantly outperforming its competitor the Nvidia H100 GPU. The key hardware specs where the MI300X surpasses the H100 are High Bandwidth Memory (HBM) capacity and bandwidth.
```

Comment:
```suggestion
## Harnessing the MI300 Superior Hardware Specs

On paper, the AMD Instinct MI300X GPU accelerators contain some of the best hardware specifications on the market, significantly outperforming its competitor the Nvidia H100 GPU. The key hardware specs where the MI300X surpasses the H100 are High Bandwidth Memory (HBM) capacity and bandwidth.
```

Reply:
moved

Reply:
I created a subsection with the more neutral wording of `## Introduction`